web/src/integration_url_modal.js

```diff
@@ -166,6 +214,20 @@ export function show_generate_integration_url_modal(api_key) {
             event.preventDefault();
             event.stopPropagation();
         }
+
+        function set_events_param(params) {
+            const $checked_check_boxes = $(".integration-event-checkbox:checked");
```

Comment:
I'd rename this to be `$selected_integration_events`. Again, I suggest avoiding variable names like this one as it's not clear when reading it what checkboxes are checked. There are more checkboxes on the modal than these specific ones.